utils/src/main/scala/za/co/absa/enceladus/utils/testUtils/TZNormalizedSparkTestBase.scala

```diff
@@ -24,7 +24,9 @@ trait TZNormalizedSparkTestBase extends SparkTestBase {
     val result = super.initSpark
 
     //TODO make conditional on empty SparkTestBase.timezone, once SparkCommons 0.3.0 will have been released
-    TimeZoneNormalizer.normalizeAll(result)
+    if (sparkConfig.timezone.isEmpty) {
```

Comment:
the comment above seems obsolete?

Reply:
Of course it is. Thank you for noticing. 👍